src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -449,6 +449,13 @@ def _satisfaction_equivalence(org_id: int, satisfaction_tag_value: str) -> Funct
     )
 
 
+def _web_vital_equivalence(org_id: int, measurement_rating: str) -> Function:
+    return Function(
+        "equals",
+        [],
```

Comment:
Did you forget to implement this?

Reply:
No I meant to push another commit to remove this.